moodle_rb.gemspec

```diff
@@ -11,12 +11,12 @@ Gem::Specification.new do |spec|
   spec.homepage      = 'https://github.com/jobready/moodle-rb'
   spec.license       = 'MIT'
 
-  spec.files         = `git ls-files -z`.split("\x0")
+  spec.files         = Dir["lib/**/*"] + %w(README.md LICENSE Rakefile)
```

Comment:
The previous `git ls-files` command gives a dubious ownership error on my system and builds an empty file.

It also seemed to attempt to include everything under the sun, so I have changed it to be more in line with our other gems, where only the files actually needed are included.